### ISSUE TITLE
:export pseudo selector not supported.

### ISSUE THREAD

**Issue by lukedar** [stylelint/stylelint#2686]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?
This selector will fail :export
Exports are useful when padding variables to from a css module to a js component.
e.g. "A bug where..."

> Which rule, if any, is this issue related to?
selector-pseudo-class-no-unknown

> What CSS is needed to reproduce this issue?

e.g.

```css
:export {
 transition: $mytransition; 
} 
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "selector-pseudo-class-no-unknown": true
  }
}
```

> Which version of stylelint are you using?

e.g. `3.10.10`

**Comment by evilebottnawi** [comment]
@lukedar your are use ICSS?

**Comment by jeddy3** [comment]
As per the docs:

> This rule considers pseudo-class selectors defined in the CSS Specifications, up to and including Editor's Drafts, to be known.

Use the [`ignorePseudoClasses: []` secondary option](https://github.com/stylelint/stylelint/tree/master/lib/rules/selector-pseudo-class-no-unknown#optional-secondary-options) if you wish to ignore `:export`.

**Comment by evilebottnawi** [comment]
@jeddy3 seems ICSS in near future gain popularity, maybe we should ignore this by default.

**Comment by jeddy3** [comment]
The ` ignorePseudoClasses` option is specifically there for methodologies and/or supersets. It's trival to make use of the option in a configuration object. Additionally, we have a _consistent_ definition for "unknown" at the moment that we should maintain.

**Comment by lukedar** [comment]
@evilebottnawi and @jeddy3 Thanks for this information.
